BCH-Wunder-Project-team-4/Wunder.io
# styling update for scrolling numbers

## Link to ticket:

#50 

## Link to feature environment:

these two:
http://localhost:3000/cases/lukefi-statistics-news-and-data-bank-one-place
http://localhost:3000/en/about-wunder/wunder-story

## Changes proposed in this PR:

<img width="1348" alt="Screenshot 2023-12-20 at 11 09 02" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/bb406031-0f92-4d53-b016-eca7b0e4d5bc">


## How to test:

1. `git fetch origin issue50Number`
2. `git checkout issue50Number`
3.  go to these links and check scrolling numbers :
 http://localhost:3000/cases/lukefi-statistics-news-and-data-bank-one-place
http://localhost:3000/en/about-wunder/wunder-story
